packages/suite/src/views/wallet/send/components/Outputs/index.tsx

```diff
@@ -86,6 +93,26 @@ const Outputs = ({ disableAnim }: OutputsProps) => {
                             ) : (
                                 <>
                                     <Row>
+                                        {account.networkType === 'ethereum' && (
```

Comment:
Move above `{output.type === 'opreturn' ? (`

I would like it closable 🙏  its annoying @Hermez-cz